lib/config/fileconf.go

```diff
@@ -496,6 +497,10 @@ type Auth struct {
 	// it here overrides defaults.
 	// Deprecated: Remove in Teleport 2.4.1.
 	DynamicConfig *bool `yaml:"dynamic_config,omitempty"`
+
+	// LicenseFile is a path to the license file. The path can be either absolute or
+	// relavive to the global data dir
```

Comment:
relative. have you discussed the actual format of the file passing with @kontsevoy